src/components/RecordsModal.vue

```diff
@@ -110,6 +111,9 @@ export default defineComponent({
         }
       }
     },
+    handleCloseModal() {
```

Comment:
No need this function since emit(close) is done directly in the button component